gst/mqtt/mqttsink.c

```diff
@@ -168,6 +345,14 @@ gst_mqtt_sink_get_property (GObject * object, guint prop_id,
 static void
 gst_mqtt_sink_class_finalize (GObject * object)
 {
+  GstMqttSink *self = GST_MQTT_SINK (object);
+
+  g_free (self->mqtt_host_address);
+  g_free (self->mqtt_host_port);
+  g_free (self->mqtt_client_handle);
+  if (self->err)
+    g_error_free (self->err);
```

Comment:
Q: this impl. generally reuses `self->err` and it's freed on finalize(). I wonder whether there's no memory leak issues.
There're some codes doing `self->err = g_error_new`, which may overwrite a previous error.

Reply:
@dongju-chae Thanks. My intention is that "Do not override the previous errors" but it is not kept at all. I will fix it now.